minio/policy.py

```diff
@@ -129,10 +148,10 @@ def _get_in_use_policy(statements, bucket_name, prefix=''):
               Policy.WRITE_ONLY: False}
 
     for s in statements:
-        resource = s['Resource']
+        resource = _get_resource(s)
```

Comment:
if `s['Resource']` would raise an exception which denotes an wrong JSON.  This approach swallows the error.